mon-pix/tests/integration/components/routes/campaigns/assessment/skill-review-test.js

```diff
@@ -297,58 +289,55 @@ describe('Integration | Component | routes/campaigns/assessment/skill-review', f
                   get: sinon.stub().returns([]),
                 },
                 campaign: {
-                  organizationName: 'Dragon & Co',
-                  customResultPageButtonUrl: 'http://www.my-url.net/resultats',
-                  customResultPageButtonText: 'Next step',
                   get: sinon.stub(),
                 },
               },
             };
 
             model.campaignParticipation.get.withArgs('isShared').returns(true);
             model.campaignParticipation.campaignParticipationResult.get.withArgs('reachedStage').returns(reachedStage);
+            model.campaignParticipation.campaignParticipationResult.get.withArgs('masteryPercentage').returns(50);
             model.campaignParticipation.campaignParticipationResult.reachedStage.get.withArgs('threshold').returns([75]);
-            model.campaignParticipation.campaign.get.withArgs('customResultPageButtonUrl').returns(['http://www.my-url.net/resultats']);
-            model.campaignParticipation.campaign.get.withArgs('customResultPageButtonText').returns(['Next step']);
-            model.campaignParticipation.campaign.get.withArgs('organizationName').returns(['Dragon & Co']);
+            model.campaignParticipation.campaign.get.withArgs('customResultPageButtonUrl').returns('http://www.my-url.net/resultats');
+            model.campaignParticipation.campaign.get.withArgs('customResultPageButtonText').returns('Next step');
+            model.campaignParticipation.campaign.get.withArgs('organizationName').returns('Dragon & Co');
+            model.campaignParticipation.get.withArgs('participantExternalId').returns('identifiant');
             this.set('model', model);
           });
 
-          it('should display the button', async function() {
+          it('should display the button with all params', async function() {
             // When
             await render(hbs`<Routes::Campaigns::Assessment::SkillReview
                 @model={{model}}/>`);
 
             // Then
-            expect(find('[href="http://www.my-url.net/resultats?stage=75"]')).to.exist;
+            expect(find('[href="http://www.my-url.net/resultats?masteryPercentage=50&externalId=identifiant&stage=75"]')).to.exist;
             expect(find('[target="_blank"]')).to.exist;
             expect(contains('Next step')).to.exist;
           });
         });
 
-        context('when the participant has finished a campaign without stages', function() {
+        context('when the participant has finished a campaign with neither stages and he has neither masteryPercentage nor participantExternalId', function() {
```

Comment:
**tought :** J'ai l'impression qu'il y a de la duplication de tests entre unitaire et integ

Par exemple : 

`when the participant has finished a campaign with neither stages and he has neither masteryPercentage nor participantExternalId`

`when there is no params` / `should return the url of the custom button`

Reply:
👍